src/main/java/mate/academy/springboot/swagger/service/mapper/DtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springboot.swagger.service.mapper;
+
+public interface DtoMapper<P, D, T> {
+    P mapToModel(T t);
+
+    D mapToDto(P dto);
```

Comment:
looks strange to me, maybe as below?
```suggestion
    P mapToModel(T requestDto);

    D mapToDto(P model);
```